inst/@sym/chol.m

```diff
@@ -66,7 +66,15 @@
   elseif (nargin > 2)
     print_usage ();
   end
-  y = python_cmd('return _ins[0].cholesky(),', x);
+
+  cmd = { 'A = _ins[0]' ...
+          'try:' ...
+          '    y = A.cholesky_decomposition()' ...
+          'except AttributeError:' ...
+          '    y = A.cholesky()' ...
+          'return y' };
```

Comment:
I see `return foo,` in some functions that return a single value, is there a preference for with or without the comma?